src/main/java/datawave/query/model/ModelKeyParser.java

```diff
@@ -51,12 +84,25 @@ public static FieldMapping parseKey(Key key, Value value) {
         // we can have attributes no matter the mapping
         attributes.addAll(Arrays.asList(StringUtils.split(new String(value.get(), StandardCharsets.UTF_8), ',')));
         
-        if (1 == colq.length) {
-            // in this case we expect model field attributes (not specific to a mapping)
+        if (0 == colq.length || (colq.length == 1 && colq[0].isEmpty())) {
```

Comment:
Your example keys above are awesome. Thanks! There's doesn't appear to be one that matches this section of the parsing though.

Reply:
oh right, I was also supporting that case at one point.  That is probably too much, I will make this one error out.